pkgs/ffigen/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 10.1.0
```

Comment:
If anyone depends on FFIgen 10 and automatically generates code in their build, we would break them. Similar to if they run pub locally and run FFIgen locally again. So maybe we should go for 11.0 instead. Then we force them to deal with their bad setup when they update to FFIgen 11 instead of the next time they update FFIgen. And it will be clear to them that upgrading FFIgen caused the issue (not some other change on their system).

(Going for 10.1 would have users automatically update, see failures and figure out that they have to pin to 10.0 if they don't want to deal with the issues at that very moment.)